### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by LIBERCOSOFT**
# Create Animals Table
## In this PR; 
- Create a database named vet_clinic.
- Create a table named animals with the id, name, date_of_birth, escape_attempts, neutered, and weight_kg columns.
- Insert some animals' data.
- Write queries for the following quests:

> Find all animals whose name ends in "mon".
[screenshot_1](https://drive.google.com/file/d/1H1HhvrapHC97m88ImzIoQiAOhWxMyXHW/view?usp=sharing)
>List the name of all animals born between 2016 and 2019.
[screenshot_2](https://drive.google.com/file/d/1cI4SdE_yc-LfrbGD3QPwrPwKoDzY_DXS/view?usp=sharing)
> List the name of all animals that are neutered and have less than 3 escape attempts.
[screenshot_3](https://drive.google.com/file/d/1R5Zk6o0GGgnYpmaOIRDNlxeeCZORlIZD/view?usp=share_link)
> List the date of birth of all animals named either "Agumon" or "Pikachu".
[screenshot_4](https://drive.google.com/file/d/1kKUiGc834bOD4j58f1wZvSV3r2v5T2JP/view?usp=share_link)
> List name and escape attempts of animals that weigh more than 10.5kg
[screenshot_5](https://drive.google.com/file/d/1NqyOcql-AIs8cQ02OcRiT7IMcEK3Q6Ry/view?usp=share_link)
> Find all animals that are neutered
[screenshot_6](https://drive.google.com/file/d/17TG2RazHI5GKHHwpScNJGrj0q8Fxtopq/view?usp=sharing)
> Find all animals not named Gabumon.
[screenshot_7](https://drive.google.com/file/d/1qsY6MxVX1EiyA7qJigQNniv9vt3Y2S8G/view?usp=sharing)
> Find all animals with a weight between 10.4kg and 17.3kg 
[screenshot_8](https://drive.google.com/file/d/1UdJltRjhEfcjdKy77QLxtdEE2SCbxQgL/view?usp=sharing)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Agumon', '2020-02-3', 0, false, 10.23);
```
Comment (sja-thedude) [inline data.sql]: - [ ] Kindly make sure that the neutered value for Agumon is true as per the requirements.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN  '2016-01-01' AND  '2019-12-31';
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered=true AND escape_attempts < 3;
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT (date_of_birth) FROM animals WHERE name='Augmon' OR name='Pikachu';
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
+-- Find all animals that are neutered
+SELECT * FROM animals WHERE neutered=true;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure that true should be in ' ' quotations.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @LIBERCOSOFT, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional readme
✔️ Good job so far
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Kolapo, 
## To Highlight 🎉
PR  has a title and summay. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**